preflight-checks/src/checks/03-docker-build.js

```diff
@@ -0,0 +1,102 @@
+import chalk from 'chalk';
+import packageJson from '../package';
+import execa from "execa";
+import findUp from 'find-up';
+
+import fs from 'fs';
+import process from 'process';
+
+const chalkVIPGo = chalk.yellow( '@automattic/vip-go' );
+const chalkPackageJson = chalk.yellow( 'package.json' );
+
+const envVariables = {
+	'VIP_GO_APP_ID': '123',
+}
+
+function executeShell( command, envVars = {} ) {
+	return execa.command( command, {
+		env: Object.assign( {}, envVariables, envVars )
+	} );
+}
+
+async function findNodeVersion() {
+	// Try to fetch a .nvmrc file
+	const nvmrcPath = await findUp('.nvmrc');
```

Comment:
This is pretty cool! But since VIP doesn't respect `.nvmrc`, we wouldn't want to push customers in that direction. Maybe a better first iteration would be a simple picker with a hard-coded list of the current LTS major versions:

```
> Please select a Node.js version:
- 16
- 14
- 12
```